pkg/resources/nifi/pod.go

```diff
@@ -476,6 +476,61 @@ func (r *Reconciler) createNifiNodeContainer(nodeConfig *v1.NodeConfig, id int32
 				},
 			},
 		},
+		{
+			Name: "NIFI_SECURITY_OIDC_CLIENT_ID",
+			ValueFrom: &corev1.EnvVarSource{
+				SecretKeyRef: &corev1.SecretKeySelector{
+					LocalObjectReference: corev1.LocalObjectReference{
+						Name: "nifi-secrets",
+					},
+					Key: "NIFI_SECURITY_OIDC_CLIENT_ID",
+				},
+			},
+		},
+		{
+			Name: "NIFI_SECURITY_OIDC_CLIENT_SECRET",
+			ValueFrom: &corev1.EnvVarSource{
+				SecretKeyRef: &corev1.SecretKeySelector{
+					LocalObjectReference: corev1.LocalObjectReference{
+						Name: "nifi-secrets",
+					},
+					Key: "NIFI_SECURITY_OIDC_CLIENT_SECRET",
+				},
+			},
+		},
+		{
+			Name: "NIFI_SECURITY_OIDC_DISCOVERY_URL",
```

Comment:
Please remove this ENV. I think again and see this can be configure using normal flow